wildfly/src/main/docker/Dockerfile-domain-development

```diff
@@ -23,10 +23,11 @@ ARG WILDFLY_VERSION
 COPY wildfly-${WILDFLY_VERSION}.tar.gz /
 
 USER root
-ARG WILDFLY_VERSION
```

Comment:
Could you explain why the `ARG WILDFLY_VERSION` is no longer necessary.

Reply:
It was defined two times on the line 22 and 26
https://github.com/kstekovi/console/blob/development-images/wildfly/src/main/docker/Dockerfile-domain-development#L22